cpp/rrtmgp_const.h

```diff
@@ -81,11 +94,76 @@ typedef FArray<char,2,yakl::memHost> charHost2d;
 
 typedef FArray<std::string,1,yakl::memHost> string1d;
 
+#ifdef RRTMGP_ENABLE_KOKKOS
+typedef FView<real*>       real1dk;
+typedef FView<real**>      real2dk;
+typedef FView<real***>     real3dk;
+typedef FView<real****>    real4dk;
+typedef FView<real*****>   real5dk;
```

Comment:
@bartgol , It looks like you're leaving the index ordering the same in the Kokkos kernels. If that's the case, then shouldn't these be `LayoutLeft`? I'm curious how you got the same results without wither `LayoutLeft` or permuting the index ordering.

Reply:
@mrnorman , see above:
```
using FView = Kokkos::View<T, Kokkos::LayoutLeft, Device>;
```

All these FViews are layout left.

Reply:
Ahh, Sorry, I missed that. Thanks.

Reply:
I think these _are_ `LayoutLeft` views. The F in front of the type name makes me thnk "FortranView", which probably means `LayoutLeft`. @jgfouca may correct me though.